src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+
+  width: 200px;
+  height: 408px;
+
```

Comment:
```suggestion
```

Reply:
isn't fixed
you shouldn't hardcode body